releng_sop/koji_block_package_in_release.py

```diff
@@ -1,19 +1,37 @@
 # -*- coding: utf-8 -*-
+"""Block packages in a release.
 
+Constructs the koji command
+
+    koji --profile=KOJI_PROFILE block-pkg RELEASE_TAG PACKAGES...
+
+KOJI_PROFILE is obtained from the environment settings.
+
+RELEASE_TAG is the "tag_release" key from the release configuration
+    corresponding to the release id.
+"""
```

Comment:
This partially overlaps with the "manual steps" section in docs.
Not sure if it's duplicity or not.